esmvalcore/_recipe/check.py

```diff
@@ -456,3 +411,57 @@ def reference_for_bias_preproc(products):
             f"{len(reference_products):d}{ref_products_str}Please also "
             f"ensure that the reference dataset is not excluded with the "
             f"'exclude' option")
+
+
+def statistics_preprocessors(settings: dict) -> None:
+    """Check options of statistics preprocessors."""
```

Comment:
this is a public function - it needs a tad better docstring

Reply:
It's not - it's only in the `_recipe` module which is not exposed to the public

Reply:
ah - my bad, cheers!